server/routes/urlshortener.js

```diff
@@ -1,20 +1,64 @@
 const { WINDOW_LOCATION_ORIGIN, PORT } = require('../config/constants');
-
 const { urlShorten } = require('../models/urlShorten');
 const { ShortenedUrlsRepository } = require('../services/shortenedUrlRepository');
 const { isUri } = require('../helper/web-url-validation');
-const { generate } = require('../middleware/shortid');
 const { IndexNotFoundError, NotFoundError } = require('../models/customErrors');
+const code = require('../helper/code');
 
 var urlRepository = new ShortenedUrlsRepository();
+
+var keys = [];
```

Comment:
Does this belong in the controller/route file? I'd rather say in a shortener helper function or something.

Reply:
Yes that kinda belongs to the TODOs will move this stuff in another PR